api/config/config.go

```diff
@@ -351,6 +352,14 @@ type PyFuncPublisherConfig struct {
 	SamplingRatioRate float64 `validate:"required" default:"0.01"`
 }
 
+// InferenceServiceDefaults captures some of the configurable defaults specific to
+// KServe inference services
+type InferenceServiceDefaults struct {
+	UserContainerCPULDefaultLimit         string  `json:"userContainerCPULDefaultLimit" default:"8"`
```

Comment:
This is the new config that we can use to specify the default CPU limit we can specify on pods deployed by Merlin. Note that this only happens when the `UserContainerCPULimitRequestFactor` is specified as `0`.

Reply:
Is it typo for default limit: `UserContainerCPULDefaultLimit`, should it be `UserContainerCPUDefaultLimit`?

Reply:
Whoa ahaha yes it's a typo! Thanks for catching it!